src/scripts/clipperUI/componentBase.ts

```diff
@@ -105,6 +105,26 @@ export abstract class ComponentBase<TState, TProps> {
 				} else if (e.which === Constants.KeyCodes.tab) {
 					// Since they are using the keyboard, revert to the default value of the outline so it is visible
 					element.style.outlineStyle = "";
+				} else if (e.which === Constants.KeyCodes.up) {
+					if (element.hasAttribute("data-" + ariaSet)) {
```

Comment:
Rather than setting an attribute whose key uniquely identifies the set, why not have a predictable attribute name and assign it a value of the set name?